backend/scripts/block_traffic.sh

```
@@ -0,0 +1,56 @@
+#!/bin/bash
+export PATH="/usr/bin:/bin:$PATH"
+
+echo "=== [BLOCK_TRAFFIC] 트래픽 차단 및 Graceful Shutdown ==="
+
+cd /opt/coffee-shout
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)